### PR TITLE
Unicef Tunisia first deliverable

### PR DESCRIPTION
**Pull request by ellykits**
Signed-off-by: Elly Kitoto <junkmailstoelly@gmail.com>

### CODE REVIEW

File: opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/fragment/AdvancedSearchFragment.java [hunk 1]
```diff
@@ -50,45 +56,96 @@ protected BaseChildAdvancedSearchPresenter getPresenter() {
...
+
+        Button scanCardQRCodeView = view.findViewById(R.id.scanCardButton);
+        scanCardQRCodeView.setText(R.string.scan_qr_code);
+        scanCardQRCodeView.setVisibility(View.VISIBLE);
+        scanCardQRCodeView.setEnabled(false);
+        scanCardQRCodeView.setOnClickListener(view1 -> {
+            if (getActivity() == null) {
+                return;
+            }
+            Utils.showShortToast(getActivity(), "Implement card support");
```
Comment (bennsimon) [inline opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/fragment/AdvancedSearchFragment.java:93]: will this be translated?

File: opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/interactor/LoginInteractor.java [hunk 2]
```diff
@@ -60,8 +61,10 @@ protected void scheduleJobsPeriodically() {
         RecurringIndicatorGeneratingJob.scheduleJob(RecurringIndicatorGeneratingJob.TAG,
                 TimeUnit.HOURS.toMinutes(6), getFlexValue(BuildConfig.DATA_SYNC_DURATION_MINUTES));
 
-        // Schedule vaccine schedules update after midnight
-        AppVaccineUpdateJob.scheduleEverydayAt(AppVaccineUpdateJob.TAG, 1, 7);
+        ArchiveClientsJob.scheduleDaily();
+
+        // Schedule daily jobs
+        AppVaccineUpdateJob.scheduleEverydayAt(AppVaccineUpdateJob.TAG, 1, 20);
```
Comment (bennsimon) [inline opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/interactor/LoginInteractor.java:67]: do you need two call for this?

**Review by bennsimon — APPROVED** [review]

File: opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/fragment/AdvancedSearchFragment.java (hunk 1 above)
Comment (ellykits) [reply opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/fragment/AdvancedSearchFragment.java:93]: It's just a placeholder will be removed completely.

File: opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/interactor/LoginInteractor.java (hunk 2 above)
Comment (ellykits) [reply opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/interactor/LoginInteractor.java:67]: The archive job runs once after login then scheduled daily starting from the next day.

**Pull request merged**